kedro-datasets/kedro_datasets/snowflake/snowpark_dataset.py

```diff
@@ -34,65 +34,66 @@ class SnowparkTableDataset(AbstractDataset):
            mode: overwrite
            column_order: name
            table_type: ''
-     ```
-     You can skip everything but "table_name" if the database and schema are
-     provided via credentials. This allows catalog entries to be shorter when
-     all Snowflake tables are in the same database and schema. Values in the
-     dataset definition take priority over those defined in credentials.
-
-     Example:
-     The credentials file provides all connection attributes. The catalog entry
-     for "weather" reuses the credentials parameters, while the "polygons" catalog
-     entry reuses all credentials parameters except for specifying a different
-     schema. The second example demonstrates the use of ``externalbrowser`` authentication.
-
-     catalog.yml:
+        ```
```

Comment:
Looks like something went a bit wrong with the formatting for this one: https://kedro--1116.org.readthedocs.build/projects/kedro-datasets/en/1116/api/kedro_datasets/snowflake.SnowparkTableDataset/#kedro_datasets.snowflake.SnowparkTableDataset

Reply:
Good catch! Just an indentation issue, fixed in 83b1577.